ldms/src/sampler/lnet_stats.c

```diff
@@ -300,15 +309,15 @@ static int config(struct ldmsd_plugin *self, struct attr_value_list *kwl, struct
 
 	rc  = create_metric_set(base);
 	if (rc) {
-		msglog(LDMSD_LERROR, SAMP ": failed to create a metric set.\n");
+		msglog(LDMSD_LWARNING, SAMP ": failed to create a metric set.\n");
```

Comment:
Why is this only a warning?

Reply:
This is a warning because it is an expected and likely recoverable failure without admin intervention; should that be demoted further to info? ldmsd may be coming up while lustre is temporarily not providing lnet data in any location; this occurs if there are no active mount points. I will review the messages that will precede or follow this one during the failure cases and see if they could be made more informative.

Reply:
My reading of the code may be incorrect, but it looks like if the source file (i.e. /proc/...) could not be opened, it fails before trying to call create_metric_set. Maybe the file is there but empty?

Reply:
No, you are correct. The change to warning is incorrect, now that the file isn't actually checked as part of create_metric_set.
I will undo that.